internal/topology/check/compatibility.go

```diff
@@ -53,15 +53,15 @@ func ObjectsAreCompatible(current, desired client.Object) field.ErrorList {
 	if currentGK.Group != desiredGK.Group {
 		allErrs = append(allErrs, field.Forbidden(
 			field.NewPath("metadata", "apiVersion"),
-			fmt.Sprintf("group cannot be changed from %v to %v to prevent incompatible changes in %v/%v",
-				currentGK.Group, desiredGK.Group, currentGK.String(), current.GetName()),
+			fmt.Sprintf("apiVersion.group cannot be changed from %q tq %q to prevent incompatible changes in the Cluster",
```

Comment:
This is a pattern I'd like to be consistent on - the apiVersion is omitted here because it's included in the path above. Would it be better to duplicate fields like this to make the actual message have more direct meaning without the path?

Reply:
```suggestion
			fmt.Sprintf("apiVersion.group cannot be changed from %q to %q to prevent incompatible changes in the Cluster",
```

Reply:
Maybe `apiVersion group` would be better to avoid the impression that we're talking about a field path (as it's not really a field path)

Reply:
Hopefully, the above pattern should be applied consistently; it applies only when we are validating a composed value which is model as a single field (and if I'm not wrong the only use case is apiVersion). 
TBH I'm not concerned about giving the impression that we're talking about a field path because message is already separated from the field path in the reported error